springwolf-plugins/springwolf-amqp-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/util/ProducerDataUtil.java

```diff
@@ -0,0 +1,41 @@
+package io.github.stavshamir.springwolf.asyncapi.util;
+
+import com.asyncapi.v2.binding.amqp.AMQPChannelBinding;
+import com.asyncapi.v2.binding.amqp.AMQPOperationBinding;
+import com.google.common.collect.ImmutableMap;
+import io.github.stavshamir.springwolf.asyncapi.types.ProducerData;
+import lombok.experimental.UtilityClass;
+
+import java.util.Collections;
+
+@UtilityClass
+public class ProducerDataUtil {
```

Comment:
I agree we should provide a convenient way to create the producer data, but I have a different idea that I prefer to implement (make ProducerData an interface, and have each plugin implement it with a builder that will map the input to the correct bindings). So please remove this file and I will add this after I merge your PR.